python-package/lightgbm/sklearn.py

```diff
@@ -592,6 +595,7 @@ def _get_meta_data(collection, name, i):
                               evals_result=evals_result, fobj=self._fobj, feval=feval,
                               verbose_eval=verbose, feature_name=feature_name,
                               categorical_feature=categorical_feature,
+                              init_model=init_model,
```

Comment:
Move according to the new order. Also, please place it at the same line as `callbacks`.